### PR TITLE
[RSDK-7562]  Add enablewebprofile to robot config

### PR DESCRIPTION
**Pull request by ale7714**
Proto update here https://github.com/viamrobotics/api/pull/497

### CODE REVIEW

**Comment by cheukt** [comment]
should we not just put the rest of the rdk changes in here too? so can test the flow end to end

**Comment by ale7714** [comment]
This is finally ready for review. 

App side for the config change https://github.com/viamrobotics/app/pull/4845

File: web/server/entrypoint.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ func (s *robotServer) createWebOptions(cfg *config.Config) (weboptions.Options,
 	if err != nil {
 		return weboptions.Options{}, err
 	}
-	options.Pprof = s.args.WebProfile
+	options.Pprof = s.args.WebProfile || cfg.EnableWebProfile
```
Comment (benjirewis) [inline web/server/entrypoint.go:198]: [q] So if we run the RDK with `-webprofile` and put `enable_webprofile: false` in the JSON config, the web profile will still be enabled?

**Review by benjirewis — APPROVED** [review]
LGTM; just a question about `-webprofile` and JSON combinations.

File: web/server/entrypoint.go (hunk 1 above)
Comment (ale7714) [reply web/server/entrypoint.go:198]: Yes that's how it currently behaves. I think that behavior makes sense as in if I add the flag explicitly I expected to just work. Happy to change it if we end up finding this to be confusing.

Comment (benjirewis) [reply web/server/entrypoint.go:198]: Cool; thanks! I think what you have now makes sense 👍🏻.

**Pull request merged**